receiver/postgresqlreceiver/scraper.go

```diff
@@ -102,6 +102,17 @@ func (p *postgreSQLScraper) scrape(ctx context.Context) (pmetric.Metrics, error)
 		}
 		databases = dbList
 	}
+	exclude := make(map[string]struct{})
+	for _, db := range p.config.ExcludeDatabases {
+		exclude[db] = struct{}{}
+	}
```

Comment:
Can we construct this once in `newPostgreSQLScraper` and save it onto the `postgreSQLScraper`?